packages/datadog_flutter_plugin/lib/src/rum/rum_configuration.dart

```diff
@@ -156,6 +156,15 @@ class DatadogRumConfiguration {
   /// Defaults to disabled (`null`).
   double? appHangThreshold;
 
+  /// Enables collection of anonymous user id across sessions.
+  ///
+  /// When enabled, the SDK generates a unique, non-personal anonymous user ID that is persisted across
+  /// app launches. This ID will be attached to each RUM Session, allowing you to link sessions
+  /// originating from the same user/device without collecting personal data.
```

Comment:
very nit: just for the spelling consistency

```suggestion
  /// Enables collection of anonymous user ID across sessions.
  ///
  /// When enabled, the SDK generates a unique, non-personal anonymous user ID that is persisted across
  /// app launches. This ID will be attached to each RUM Session, allowing you to link sessions
  /// originating from the same user/device without collecting personal data.
```